.github/workflows/scripts/build_and_install.sh

```diff
@@ -4,7 +4,7 @@ set -e
 echo "Running Build & Install"
 distro="$(lsb_release -cs)"
 
-cd /build/whistleblowing-software
+cd /build/globaleaks-whistleblowing-software
```

Comment:
This file has been edited across multiple commits with instances of code being removed and then re-added. To streamline the review, the offending commits should be removed, either by rebasing or squashing them into a single, meaningful commit.